nestor_api/config/k8s.py

```diff
@@ -2,11 +2,15 @@
 import os
 
 
-# pylint: disable=too-few-public-methods
 class K8sConfiguration:
     """k8s configuration"""
 
     @staticmethod
     def get_http_proxy() -> str:
         """Returns the k8s http proxy"""
         return os.environ["NESTOR_K8S_HTTP_PROXY"]
+
+    @staticmethod
+    def get_templates_dir() -> str:
+        """Returns the name of the config sub-folder in which to look for templates."""
```

Comment:
I am pretty sure this sentence is not correct english but I am not sure about how it should be actually 😅
Maybe:
`Returns the config sub-folder's name in which looking for templates.`

Reply:
What about ?
```
Returns the name of the folder holding the templates. It will be resolved under the `config` path.
```

Reply:
Ahah actually I just red back the name of the function and it is definitely more explicit than any of those description we suggested 😅

Reply:
Then "Get directory containing Kubernetes templates" would be probably the clearest solution :)